src/main/api/2/studio-api-2.yaml

```diff
@@ -3215,6 +3217,37 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /security/encrypt:
+    get:
+      tags:
+        - security
+      summary: Encrypt a text value
+      operationId: encrypt
+      parameters:
```

Comment:
Don't we need a description with the required permission, like `description: 'Required permission "read_configuration"'`?

Reply:
Right now it doesn't require any special permission, just to be authenticated